examples/security-token/README.md

```diff
@@ -0,0 +1,91 @@
+- *permissions.py*: This is the rules smart contract. In this template we have two rules:
+  - An account cannot hold more than 100 tokens
+  - Both from, to accounts must be whitelisted
+- *clear_state_program.py*: clear state program (returns 1)
+
+`assets/asa.yaml` - This the the asset definition file.
+
+### Setup
+
+Deployment scripts are placed directly in `/scripts`:
+
+**NOTE:** For transactions involving a multisig account (eg. token can be created and issued by a multisig), user will control one of the multisig accounts and he will receive a signed transaction i.e a signed `deployASA` tx by accounts in msig <= threshold. User can append his/her using `algob sign-multisg` command & then use `executeSignedTxFromFile` function to successfully send transaction to network (eg. deploy token).
+
+1. *0-setup-token.js*: In this script we deploy the token (as an Algorand Standard Asset) using `deployer.deployASA(..)` function. Check note above if creator is a multisig address.
+NOTE: In the template
+2. *1-setup-apps.js*: In this script we first deploy `controller.py` smart contract (which saves the token_id & sets kill_status to false by default). Only token manager can deploy this contract. Secondly, we deploy the `permissions.py` smart contract, which has the rules. During permisssions deployment, we save the controller_app_id in it's global state.
+3. *2-asset-clawback.js*: Here we deploy the contract `clawback-escrow.py` and add some funds to it. In contract we save token_id, controller_app_id. After that we update the asset clawback to the address of the contract account (using algob ModifyAsset transaction)
+
+### Execution
+
+1. *Issuance* (`/scripts/issuance/issue.js`): Issuer can issue/mint few tokens to an account. Receiver will need to opt-in to the token first. For issuance tx, we need a transaction group of two transactions :-
+a) Call to the controller smart contract - it checks if token is not killed (as issuance is not allowed if token is killed).
+b) Asset transfer transaction from issuer -> receiver using clawback. `clawback-escrow.py` does the basic sanity checks for issuance_tx.
+We don't need additional rule checks as issuer sets the rules himself.
+
+2. *Update Asset Reserve* (`/scripts/issuance/update-reserve.js`): Say if reserve is a multisig address and we want to kick out one address from the multisig, then we will need to update the asset reserve address. Also, if we update the asset reserve, we will need to move all tokens from previous reserve -> new reserve. This can be done in an atomic transaction group of 2 txns:
+a) Asset transfer transaction using clawback escrow where we move all tokens from previous reserve to new one.
+b) Asset Modification transfer where we update the asset reserve address to new one.
```

Comment:
Yes, added support for rekying in #296. Will update this PR after it is merged. 
However, i would like to keep this approach as well (maybe commented), because with rekeying we update an account itself (rather than a property of asset).

Reply:
OK. Then let's implement one and add a note for the alternative solution (we can even implement both).

Also, in the text above, do we need word "transfer" in "Asset Modification transfer"?

Reply:
> OK. Then let's implement one and add a note for the alternative solution (we can even implement both).

I implemented both approaches in a separate functions - [https://github.com/scale-it/algo-builder/pull/292/files#diff-906cbbe7662016ad6886d22745796565b21389c606b587ac4f559bb352a366abR137](https://github.com/scale-it/algo-builder/pull/292/files#diff-906cbbe7662016ad6886d22745796565b21389c606b587ac4f559bb352a366abR137)

> Also, in the text above, do we need word "transfer" in "Asset Modification transfer"?

No, it was a typo. Updating to "Asset Modification transaction"